src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,13 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+div {
+  margin: 0;
+}
+
+html {
```

Comment:
I think it's better to write styles for html first but idk if it is important

Reply:
I think it`s not important, but who knows 👀